src/app/configuration/environments/edit-environment/edit-environment.component.html

```diff
@@ -371,9 +387,23 @@ <h4>Environment Specifics</h4>
         </tr>
       </thead>
       <tbody>
-        <tr *ngFor="let item of env.environment_specifics | keyvalue">
-          <td>{{ item.key }}</td>
-          <td>{{ item.value }}</td>
+        <tr *ngFor="let item of env.environment_specifics | keyvalue">         
+          <!-- view of unedit value -->
+          <td *ngIf="uneditedEnv.environment_specifics.hasOwnProperty(item.key)">{{ item.key }}</td>
```

Comment:
i would also use `<ng-container>` here, you and put both corresponding `td` inside it. 
It is hard to get that the `item.key` is used for both unedited and edited values